tests/textwatcher.js

```diff
@@ -40,12 +40,25 @@ describe( 'TextWatcher', () => {
 
 	afterEach( () => {
 		sinon.restore();
+		watcher.off();
```

Comment:
Shouldn't be necessary AFAICS.

After each test, a new instance is created and old references should be released.